mautic/mautic
# [UX] Updating Blank theme to MJML

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR updated Blank from HTML to MJML.




| Before                                 | After
| -------------------------------------- | ---
| ![image](https://github.com/user-attachments/assets/fad2abdb-a415-47f6-b14a-4187bbce8264)                                       | ![image](https://github.com/user-attachments/assets/7f6dd67c-d79f-4aa2-b72f-32aa79e43a6a)


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Emails
3. Create new
4. Select blank
5. Open the editor
6. See they're very similar
